doc/decision-tree-widget/tree.yml

```diff
@@ -117,6 +117,9 @@ children:
       - label: and output the computed values as a nested Observable when the source emits a value
         children:
         - label: mergeScan
+      - label: and output the computed values as a nested Observable when the source emits a value and cancel the previous nested Observable
```

Comment:
Since you opened this, the decision tree widget has moved to `docs_app\tools\decision-tree-generator\src\tree.yml`, so this change should be applied to that file.